### PR TITLE
Create Activity Feeds topics

### PR DESCRIPTION
**Pull request by diana-lakatos**
1 concept topic and 1 tutorial for the How-To Guides/Activity Feeds section.

### CODE REVIEW

File: marketplace_builder/views/pages/how-to/activity-feeds/implementing-activity-feed.liquid [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+1. Create activity
+2. Publish activity
+3. Display feed
+4. Get all feed names 
+
+### Step 1: Create activity 
+Prepare an activity payload (hash). 
+
+{% include 'alert/note', content: 'The activity payload can have any structure, but it's recommended to follow the <a href="https://www.w3.org/TR/2017/REC-activitystreams-core-20170523/">W3C Recommendation for Activity Streams 2.0</a>.' %}
```
Comment (pavelloz) [inline marketplace_builder/views/pages/how-to/activity-feeds/implementing-activity-feed.liquid]: '

File: marketplace_builder/views/pages/how-to/activity-feeds/implementing-activity-feed.liquid [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+Your activity has been published to `UserNotifications[user_id]` feed. 
+
+### Step 3: Display feed
+To read the content of your feed, you have to provide `feed_name` and `feed_id`: 
+
+```liquid
+{% raw %}
+query activities($type: String!, $id: ID!, $per_page: Int, $page: Int) { streams(feed_id: $id, feed_type: $type, per_page: $per_page, page: $page) { total_entries results { id uuid payload } } }
+
+{% query_graph activities, result_name: g, type: 'UserNotifications', id: user_id %}
```
Comment (pavelloz) [inline marketplace_builder/views/pages/how-to/activity-feeds/implementing-activity-feed.liquid]: `result_name: 'g',`

File: marketplace_builder/views/pages/how-to/activity-feeds/implementing-activity-feed.liquid [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+```
+
+Your activity has been published to `UserNotifications[user_id]` feed. 
+
+### Step 3: Display feed
+To read the content of your feed, you have to provide `feed_name` and `feed_id`: 
+
+```liquid
+{% raw %}
+query activities($type: String!, $id: ID!, $per_page: Int, $page: Int) { streams(feed_id: $id, feed_type: $type, per_page: $per_page, page: $page) { total_entries results { id uuid payload } } }
```
Comment (pavelloz) [inline marketplace_builder/views/pages/how-to/activity-feeds/implementing-activity-feed.liquid]: Some line breaks would help
+ this line should be graphql formatted

**Comment by diana-lakatos** [comment]
@pavelloz Thank you for your review! Fixed all.

**Comment by pavelloz** [comment]
Fixed errors/typos - its always a good idea to sync it somewhere to fix things like `grapql` => `graphql` they are obvious when seen, but hard to see in code review

**Comment by pavelloz** [comment]
BTW. Its missing in nav so ill wait with deploy

**Pull request merged**